meta-hovergames-distro/recipes-hovergames/images/imx-image-hovergames-demo.bb

```diff
@@ -532,11 +533,25 @@ do_fix_connman_conflict() {
 	set +x
 }
 
+
+#We need to add Yocto libraries to LD path and remove conflicting libraries
 fakeroot do_fix_ldconfig() {
 	#Ld config mises /usr/lib path
 	set -x
 
 	echo >>"${APTGET_CHROOT_DIR}/etc/ld.so.conf.d/01-yocto.conf" "/usr/lib"
+	rm ${IMAGE_ROOTFS}/usr/lib/libatk*
+    rm ${IMAGE_ROOTFS}/usr/lib/libgtk*
+    rm ${IMAGE_ROOTFS}/usr/lib/libgudev*
+    rm ${IMAGE_ROOTFS}/usr/lib/libgdk*
+    rm ${IMAGE_ROOTFS}/usr/lib/libcairo*
+    rm ${IMAGE_ROOTFS}/usr/lib/libepoxy*
+    rm ${IMAGE_ROOTFS}/usr/lib/libpango*
+    rm ${IMAGE_ROOTFS}/usr/lib/libpixman*
+    rm ${IMAGE_ROOTFS}/usr/lib/libpng*
+    rm ${IMAGE_ROOTFS}/usr/lib/libfreetype*
+    rm ${IMAGE_ROOTFS}/usr/lib/libfont*
+    rm ${IMAGE_ROOTFS}/etc/fonts/fonts.conf
```

Comment:
Can these be removed instead in ubuntu-base using `RCONFLICTS/RREPLACES`?

Reply:
I've tried it but `RCONFLICTS/RREPLACES` works well for binaries but not libraries.